website/pages/en/_meta.js

```diff
@@ -3,19 +3,24 @@ export default {
   '---1': {
     type: 'separator',
   },
+  '###1': {
+    type: 'heading',
+    title: 'Overview',
+  },
   about: '',
   network: 'The Graph Network',
-  sunrise: 'Post-Sunrise Upgrade FAQ',
-  billing: '',
-  glossary: '',
   tokenomics: 'Tokenomics',
-  arbitrum: 'Scaling with Arbitrum',
+  contracts: 'Protocol Contracts',
+  glossary: '',
+  explorer: '',
+  firehose: '',
+  substreams: '',
   '---2': {
     type: 'separator',
   },
-  '###1': {
+  '###2': {
     type: 'heading',
-    title: 'Subgraphs',
+    title: 'Developers',
```

Comment:
Hmm, interesting change. Seems a bit strange to have a "Developing" section under a "Developers" heading, haha. Are those sections currently not all about subgraphs?

Reply:
Also a couple of those section titles should probably be changed to include the word "subgraphs" now that it's not in the heading, like "Managing" and maybe "Deploying" and "Publishing" too.